README.md

```diff
@@ -64,40 +54,22 @@ If you just want to run the migrations but leave the data intact, you can just d
 
 If you would like to interact directly with the database
 
-- you can use pgAdmin to view the database and tables, run `devstack up pgadmin` and view it in the browser at: [`http://localhost:5051/`](http://localhost:5051/)
-- Alternatively, you can dowload a tool like [PSequel](http://www.psequel.com/)
-
-### Seeding the LDAP server
-
-The task `mix meadow.ldap.setup [seed_file ...]` will seed the LDAP database using one or more LDIF files. `mix meadow.ldap.teardown [seed_file ...]` will remove any entries referenced in the seed files.
-
-- Seed files containing a reasonable sample of users and groups for development are available from the NUL dev team.
-- A seed file for testing is included in the project and is loaded automatically as part of the `mix test` task.
-
 ### Run the Elixir test suite
 
 - Start test devstack: `devstack -t up meadow`
 - run `mix test`
 
-### Accessing S3 Buckets in Development
-
-[Localstack](https://localstack.cloud), which we use to emulate Amazon S3 (and other services) in the development environment, does not have a full-featured S3 web UI, so it's necessary to use a utility like the [LocalStack AWS Command Line Interface](https://github.com/localstack/awscli-local) or [Cyberduck](https://cyberduck.io). There is a [Cyberduck Shortcut](Localstack%20S3.duck) in the root of this repository that will configure the app the connect to the S3 dev instance.
-
-*Note*: You may receive “incomplete transfer” warnings when uploading files through Cyberduck. This appears to be a bug in Cyberduck, and can be safely ignored. If you click the refresh icon in the main window after upload, you should see the uploaded files.
-
-
 ### GraphQL API
 
-You can visit the GraphiQL interface at: [`https://devbox.library.northwestern.edu:3001/api/graphiql`](https://devbox.library.northwestern.edu:3001/api/graphiql)
+You can visit the GraphiQL interface at: [`https://[YOURENV].dev.rdc.library.northwestern.edu:3001//api/graphiql`](https:/[YOURENV].dev.rdc.library.northwestern.edu:3001/api/graphiql)
 
-### IIIF Server
+### Opensearch Dashboard
 
-In dev mode, the IIIF Server is available at: `http://localhost:8183/iiif/2`
+- To start: `es-proxy start` (if you get an error, run `es-proxy reset` and then run the start again)
```

Comment:
`es-proxy reset` is no longer required. The `es-proxy` script has been updated to detect the running container instead of using a semaphore file that might get orphaned.